src/neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -53,31 +53,31 @@ public bool IsBlocked(DataCache snapshot, UInt160 account)
             return snapshot.Contains(CreateStorageKey(Prefix_BlockedAccount).Add(account));
         }
 
-        [ContractMethod(CpuFee = 1 << 15, RequiredCallFlags = CallFlags.WriteStates)]
+        [ContractMethod(CpuFee = 1 << 15, RequiredCallFlags = CallFlags.States)]
```

Comment:
And these simple setters can probably also work fine with write only. Although regular contract couldn't do that without read permission just because `GetContext` requires it.

Reply:
`CheckCommittee()` needs to call `GetCommittee()`, which needs `ReadStates`.